### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by DaniilManakov**
[DEMO LINK](https://daniilmanakov.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://daniilmanakov.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,8 +11,52 @@
...
-    <h1>Product cards</h1>
+    <div
+      class="block"
+      data-qa="card"
+    >
+      <div class="block__img"></div>
+      <div class="block__content">
+        <div class="block__text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
```
Comment (nazarmatsevych) [inline src/index.html:36]: Use semantic tags for text content like `<h1>`, `<h2>`, `<p>` and so on

File: src/index.html [hunk 2]
```diff
@@ -11,8 +11,52 @@
...
+        <div class="block__under-text">Product code: 195434</div>
```
Comment (nazarmatsevych) [inline src/index.html:37]: same here

File: src/index.html [hunk 3]
```diff
@@ -11,8 +11,52 @@
...
+        <div class="block__review">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="block__review-text">Reviews: 5</div>
```
Comment (nazarmatsevych) [inline src/index.html:46]: and here, use semantic tags

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,127 @@
+$main-accent: #060b35;
+$secondary: #616070;
```
Comment (nazarmatsevych) [inline src/styles/index.scss:2]: Create a separate file for your variables

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,127 @@
...
+    display: flex;
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+
+  }
+}
+
+
+.stars {
```
Comment (nazarmatsevych) [inline src/styles/index.scss:112]: Create a separate file for each bem block

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Good work, please fix the comment below and try to remove backstop_data folder and rerun your tests

File: src/styles/index.scss (hunk 5 above)
Comment (DaniilManakov) [reply src/styles/index.scss]: Hi, I'm getting an error on import

**Pull request closed without merging**